firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+WebViewDelegates.swift

```diff
@@ -989,7 +1023,9 @@ extension BrowserViewController: WKNavigationDelegate {
 
     func webView(_ webView: WKWebView, didFinish navigation: WKNavigation?) {
         webviewTelemetry.stop()
-
+        if isPDFRefactorEnabled {
+            (webView as? TabWebView)?.removeDocumentLoadingView()
```

Comment:
Could we cast in the `if` instead? I'm less of a fan of the syntax `(webView as? TabWebView)?.`